flexget/plugins/metainfo/imdb_lookup.py

```diff
@@ -403,7 +404,8 @@ def lookup(self, entry, search_allowed=True, session=None):
                 log.exception(e)
             raise plugin.PluginError('Invalid parameter: %s' % entry['imdb_url'], log)
 
-        for att in ['title', 'score', 'votes', 'year', 'genres', 'languages', 'actors', 'directors', 'writers', 'mpaa_rating']:
+        for att in ['title', 'score', 'votes', 'year', 'genres', 'languages', 'actors', 'directors', 'writers',
+                    'mpaa_rating']:
             log.trace('movie.%s: %s' % (att, getattr(movie, att)))
```

Comment:
I'm only trying to fix warnings from flake8, other fixes are out of scope for this set of changes.

Is there a list of best practices or alternatively old practices that should be changed?  That would be useful for contributors.

Thanks,